examples/apfl_example/config.yaml

```diff
@@ -1,5 +1,6 @@
 # Parameters that describe server
 n_server_rounds: 25 # The number of rounds to run FL
+n_warm_up_rounds: 5 # The number of rounds to run before starting APFL
```

Comment:
I would say that we don't want to mix the configs here across separate runs. That is, the warm up training should have its own configuration rather than sharing one.